dts/arm/nordic/nrf5340_cpuapp_peripherals.dtsi

```diff
@@ -421,3 +421,16 @@ timer2: timer@11000 {
 	prescaler = <0>;
 	label = "TIMER_2";
 };
+
+usbd: usbd@36000 {
```

Comment:
Add also a corresponding `/delete-node/` entry in `nrf5340_cpuapp_qkaa_eng_a.dtsi` (see https://github.com/zephyrproject-rtos/zephyr/commit/619a498845fa33fb830215b4caca21d8438a5d90).

@ioannisg I think in #26110 we missed that those nodes for non-functional peripherals should be deleted also in the non-secure part, i.e. `nrf5340_cpuappns_qkaa_eng_a.dtsi`. Could this be addressed in this PR or do you prefer to have it done in a separate one?

Reply:
For now I updated only secure secure part.